ui/packages/components/src/icons/frameworks/Django.tsx

```diff
@@ -1,8 +1,14 @@
 export function IconDjango({ className, size = 18 }: { className?: string; size?: number }) {
```

Comment:
I get the idea of passing a size property but this is very different from our implementation in other icons. All our icons are receiving tailwind classes. It would be more interesting to even create a custom tailwing configuration that would match icon sizes with text size classes

Reply:
We can tackle that later. This PR is just about bringing icon colors inline with designs